addons/headless/CfgEden.hpp

```diff
@@ -0,0 +1,35 @@
+class Cfg3DEN {
+    class Object {
+        class AttributeCategories {
+            class ace_attributes {
+                class Attributes {
+                    class GVAR(blacklist) {
+                        displayName = "Headless Blacklist";
+                        tooltip = "If blacklisted this vehicle will not be transfered to headless clients.";
+                        property = QGVAR(blacklist);
+                        control = "Checkbox";
+                        condition = "objectVehicle";
```

Comment:
This will only show on vehicles that can take crew. Altogether it only shows when editing a group or vehicle, not single infantry units (as I assume you can't blacklist single units in a group?).